lib/utility/es_client.rb

```diff
@@ -34,6 +35,9 @@ def connection_configs
       else
         raise 'Either elasticsearch.cloud_id or elasticsearch.hosts should be configured.'
       end
+      configs[:log] = es_config[:log] || false
+      configs[:trace] = es_config[:trace] || false
+      configs[:logger] = es_config[:logger] || ::Logger.new(IO::NULL)
```

Comment:
Should add to [connectors-ruby/config/connectors.yml.example](https://github.com/elastic/connectors-ruby/blob/main/config/connectors.yml.example)

Why not default to `Utility::Logger.logger`?

Reply:
Because we want to silence the Elasticsearch lib logger

Reply:
oh but the way, logger is an object, so we won't surface this option to the config. here the intent is just to pass a null logger

Reply:
Then why not just set `config[:log] = false`, if we are not gonna log it anywhere?

Reply:
Because as a developer, I want to have the ability to set the Elasticsearch logger to trace and log if I need to debug something.  For `logger` we could set it to `Utility::Logger.logger` I guess *IF* log or trace are set to true.
And if both are false, just use ::Logger.new(IO::NULL)